adc_export.py

```diff
@@ -51,11 +52,12 @@ def main(get_config,
             log.info('Records: %s', records)
             for record_chunk in chunks(records, chunk_size):
                 log.info('Chunk: %s to %s', record_chunk[0], record_chunk[-1])
+
                 data = data_proj.export_records(records=record_chunk,
                                                 format=file_format,
                                                 forms=[row['formname'], ],
-                                                fields=field_names,
```

Comment:
Can you give me directly link to the exported csv(s)  ? And does it have expected output?
example:

http://sable:8080/job/ADC_multi_project_export/xyz_1.csv
http://sable:8080/job/ADC_multi_project_export/xyz_2.csv

Reply:
http://sable:8080/job/ADC_multi_project_export/ws/

Yes from a scan everything is being displayed good in csv excels